modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/META-INF/resources/notification/form_entry_add_body.soy

```diff
@@ -54,7 +53,7 @@
 							<div id="introduction" style="background-color: #fff; border-radius: 4px; margin: 0 auto 24px auto; padding: 40px;">
 							<h1 align="center" style="margin-bottom: 24px; margin-top: 0;">{$siteName}</h1>
 
-							<h2 align="center" style="margin-bottom: 24px; margin-top: 0;">Hi, {$authorName}</h2>
```

Comment:
@linolaoj, please keep the comma. In addition, please visit the class `DDMFormEmailNotificationSender` and remove the `authorName` from there too.

Thanks.